docs/contributor/06-03-lifecycle-management.md

```diff
@@ -0,0 +1,5 @@
+# Lifecycle management of Keda Manager in Kyma
+
+When you enable the Keda module using your Kyma runtime Kyma custom resource (CR), the Lifecycle Manager (LM) downloads the bundled package of the Keda Manager and installs it. Additionally, it applies a sample Keda CR, which triggers Keda Manager to install the Keda module.
+
+![Enable Keda module with LM](/docs/assets/keda-lm-overview.drawio.svg)
```

Comment:
(1) User enables Keda module in Kyma Custom Resource
(2) Lifecycle manager reads module template of Keda module
(3) Lifecycle deploys keda manager ( using artefacts from module template)
(4) Lifecycle manager applies default Keda Custom Resource ( from module template)
(5) Keda manager watches Keda CR
(6) Keda manager reconciles Keda workloads
(7) User may change configuration by changing Keda CR spec (Keda Manager will reconcile accordingly)